botocore/httpsession.py

```diff
@@ -97,6 +98,17 @@ def create_urllib3_context(ssl_version=None, cert_reqs=None,
     return context
 
 
+def ensure_boolean(val):
```

Comment:
Yeah, we import all of httpsession into utils for the IMDS Fetchers so we're unable to import anything back into here. In the long term, I'd prefer we just move IMDSFetchers out to their own module since they're more core infrastructure than "utils" now.